site/en/adminGuide/tls.md

```diff
@@ -0,0 +1,456 @@
+---
+id: tls.md
+title: Encryption in Transit
+summary: Learn how to enable TLS proxy in Milvus.
+---
+
+# Encryption in Transit
```

Comment:
1. Both client-to-server and server-to-server TLS are supported.
2. Yes, but whether it's standalone or cluster does not affect the process. Users only need to change the value of `tlsEnabled` to true.
3. No requirement for the version.

Reply:
From Fu Bang, "No (no need to mention both server-to-server and client-to-server are supported), because the server-to-server can be also seen as the client-to-server"